src/shmem_team.c

```diff
@@ -299,7 +299,8 @@ int shmem_internal_team_split_strided(shmem_internal_team_t *parent_team, int PE
     }
 
     int global_PE_start = shmem_internal_team_pe(parent_team, PE_start);
-    int global_PE_end   = global_PE_start + PE_stride * (PE_size -1);
+    int global_PE_stride = parent_team->stride * PE_stride;
```

Comment:
Do we need to take the absolute value?

Reply:
I don't think so... it can help to think of an example.

Let's assume `parent_team` is a team that was split from team world with a negative stride, and `PE_stride` is positive.

Say, `world = {0,1,2,3}` and we split with (start,stride,size) of (3,-1,4) so `parent_team = {3,2,1,0}`.  Now we do another simple duplication split of `parent_team` with (0,2,2).  In this case we want `new_team = {3,1}`.

For the duplication split, since `parent_team->stride` is negative (it's the global stride, -1), and `PE_stride` is positive 2, we want the global stride to remain negative.  In this case -2.  With absolute value, the result above would lead to a "team wraparound" in the positive direction.

Does that makes sense?

Reply:
Thanks @davidozog. This definitely helps. I was thinking the example differently, I guess. 
For example, in your case, if the 2nd split is chosen to be (3, -2, 2) on parent team {3, 2, 1, 0} to get a team {2,0}. Then because we are not taking the absolute value, the `global_pe_stride` would be positive, which would cause wrap-around, right? Did I miss something?

Reply:
I think if the 2nd split is chosen to be (3, -2, 2) on parent team {3, 2, 1, 0}, this would actually result in team {0,2}, which has a positive stride relative to the global team.

So indeed two negative strides indeed make a positive stride.  Let me think if I can prove this more formally...